idl/chimera.pro

```diff
@@ -0,0 +1,535 @@
+;+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
```

Comment:
Preferably not, I know it would be a god practice to separate off some of the code into other functions however the algorithm should be in one neat bundle for ease of use. Personally whenever I open a coding package with tonnes of small functions I feel very overwhelmed and hate having to find the main procedure and see when it links into which function, which links to another function within it. All in all that method frustrates me. I'd rather it all be linear for an individual algorithm, especially when it is simple.